fixtures/garagecube/sfx-01-60-white.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "SFX-01 LED Bar 1m Pure White",
+  "categories": ["Pixel Bar"],
+  "meta": {
+    "authors": ["Matthieu Beghin"],
+    "createDate": "2025-09-05",
+    "lastModifyDate": "2025-09-05"
+  },
+  "links": {
+    "manual": [
+      "https://www.garagecube.com/download/manual-sfx01-50cm-2/"
```

Comment:
This leads to a download page, but unfortunately, the manual is only about the physical construction.  It doesn't say anything about DMX control.